frontend/src/components/CommentList/CommentList.test.jsx

```diff
@@ -30,14 +30,20 @@ const fakeList = [
 
 describe('User comment list', () => {
   it('Should properly render component', async () => {
+    //given
     render(<CommentList comments={fakeList} />);
+    //when
     const title = screen.getByText('Comments');
+    //then
     expect(title).toBeInTheDocument();
```

Comment:
Prawie super, tylko oddzielaj spacjami.
Co tutaj sprawdzasz? Czy się renderuje. Czyli render jest w `when`.
A `given` to są dane początkowe, pewne, takie pewniaki. W tym przypadku nie masz takich.

```suggestion
    //when
    render(<CommentList comments={fakeList} />);
   
    //then
     const title = screen.getByText('Comments');
    expect(title).toBeInTheDocument();
```